src/main/resources/swagger/api.yml

```diff
@@ -670,6 +670,7 @@ definitions:
   Attachment:
     type: object
     description: 'Complements the description of an element (for instance a product) through video, pictures...'
+    discriminator: attachmentType
```

Comment:
this is the only way how to achieve polymorphism in openapi

https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/

Reply:
@vkirichenko why do not use `@type` field as discriminator?
```
'@type':
        type: string
        description: The class type of the actual resource (for type extension).
```

seems this is it designed for.

Reply:
@EvgeniyMurenko let's check and discuss this

Reply:
@amedvedchuk done